src/services/form.js

```diff
@@ -0,0 +1,128 @@
+const httpStatusCode = require('@generics/http-status')
+const formQueries = require('@database/queries/form')
+const utils = require('@generics/utils')
+const KafkaProducer = require('@generics/kafka-communication')
+const form = require('@generics/form')
+const responses = require('@helpers/responses')
+
+module.exports = class FormsHelper {
+	/**
+	 * Create Form.
+	 * @method
+	 * @name create
+	 * @param {Object} bodyData
+	 * @returns {JSON} - Form creation data.
+	 */
+
+	static async create(bodyData, orgId) {
+		try {
+			const form = await formQueries.findOne({ type: bodyData.type, organization_id: orgId })
+			if (form) {
+				return responses.failureResponse({
+					message: 'FORM_ALREADY_EXISTS',
+					statusCode: httpStatusCode.bad_request,
+					responseCode: 'CLIENT_ERROR',
+				})
+			}
+			bodyData['organization_id'] = orgId
+			await formQueries.create(bodyData)
+			await utils.internalDel('formVersion')
+			await KafkaProducer.clearInternalCache('formVersion')
```

Comment:
kafka consumer and producer is not there please check this file from mentoring 
https://github.com/ELEVATE-Project/mentoring/blob/release-2.6.1/src/generics/kafka-communication.js
https://github.com/ELEVATE-Project/mentoring/blob/release-2.6.1/src/configs/kafka.js

@priyanka-TL @adithyadinesh0412